### PR TITLE
ACRS-57 Implement /children

### PR DESCRIPTION
**Pull request by michaelhartddat**
## What? 

[ACRS-57 Implement /children](https://collaboration.homeoffice.gov.uk/jira/browse/ACRS-57)
Add the `/children` page to the `acrs/` steps

## Check list

Checked

### CODE REVIEW

File: apps/acrs/translations/src/en/pages.json [hunk 1]
```diff
@@ -142,7 +142,10 @@
     "header": "SKELETON: /partner-summary"
   },
   "children": {
-    "header": "SKELETON: /children"
+    "header": "Children aged under 18",
+    "paragraph-1": "You can refer your own children to come to the UK, but they must have been born on or after 28 August 2003.",
+    "paragraph-2": "The children you refer must not be living independently or have their own family.",
+    "question-1": "Are you referring your children to come to the UK?"
```
Comment (jamiecarterHO) [inline apps/acrs/translations/src/en/pages.json:148]: I think in this case this text should be the 'legend' in the translations/fields file and should appear automatically without needing to render it in the view as a question.

See this: https://github.com/UKHomeOffice/acrs/blob/master/apps/acrs/fields/index.js#L228
and this: https://github.com/UKHomeOffice/acrs/blob/master/apps/acrs/translations/src/en/fields.json#L222

File: apps/acrs/views/children.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  {{$page-content}}
+    <p>{{#t}}pages.children.paragraph-1{{/t}}</p>
+    <p>{{#t}}pages.children.paragraph-2{{/t}}</p>
+    <p class="bold">{{#t}}pages.children.question-1{{/t}}</p>
+    {{#fields}}
+      {{#renderField}}{{/renderField}}
+    {{/fields}}
+    {{#input-submit}}continue{{/input-submit}}
+  {{/page-content}}
+{{/partials-page}}
```
Comment (jamiecarterHO) [inline apps/acrs/views/children.html:11]: Pop a newline at the end of this file to remove this warning sign

**Review by jamiecarterHO — CHANGES_REQUESTED** [review]
I think you can switch where some of the added copy is rendered to follow convention a little more.

File: apps/acrs/translations/src/en/pages.json (hunk 1 above)
Comment (michaelhartddat) [reply apps/acrs/translations/src/en/pages.json]: Yes that's a better approach. I had tried a few combos but couldn't quite get it.

**Review by jamiecarterHO — APPROVED** [review]
Looks good

**Pull request merged**